app/src/main/java/com/example/poste/http/PostRequest.java

```diff
@@ -4,9 +4,9 @@ public class PostRequest {
     private String title;
     private String description;
     private String url;
-    private int folder;
+    private String folder;
 
-    public PostRequest(String title, String description, String url, int folder) {
+    public PostRequest(String title, String description, String url, String folder) {
```

Comment:
You mean String folderID?

Reply:
I only changed the type here so we weren't casting to and from strings and ints; this should match what the backend expects to receive though, which appears to be `folder_id`. Just had to scope out what I was covering in one PR

Reply:
I have refactored the name to `folder_id` as this is what the serializer on the backend expects, I believe.